scripts/test/kubernetes-setup.sh

```diff
@@ -5,8 +5,28 @@
 set -x
 set -e
 
-pip install --force-reinstall 'dbt-postgres>=1.8'
-pip install --force-reinstall dbt-adapters
+DBT_VERSION="$1"
+echo "DBT_VERSION:"
+echo "$DBT_VERSION"
+NEXT_MINOR_VERSION=$(echo "$DBT_VERSION" | awk -F. '{print $1"."$2+1}')
+
+pip uninstall dbt-adapters dbt-common dbt-core dbt-extractor dbt-postgres dbt-semantic-interfaces -y
+pip install -U \
+  "dbt-core==$DBT_VERSION" \
```

Comment:
we may not need this since dbt-postgres on the next line will install dbt-core, I guess.

Reply:
We actually need this. One of the problems was that `dbt-postgres==1.9` was installing `dbt-core==1.10` :(